pythran/pythonic/types/long.hpp

```diff
@@ -87,14 +87,23 @@ namespace pythonic
     void *storage = ((boost::python::converter::rvalue_from_python_storage<
                          mpz_class> *)(data))->storage.bytes;
     auto s = PyObject_Str(obj_ptr);
+#if PY_MAJOR_VERSION >= 3
+    new (storage) mpz_class(PyUnicode_AS_DATA(s));
```

Comment:
How can we still have mpz as input? You said there are no difference between int and long in Python3.